scripts/prow-smoke-test.sh

```diff
@@ -33,12 +33,11 @@ DISTRO=$2
 PRODUCT_NAME=$3
 VERSION=$4
 ARCH=$(uname -m)
-if [[ $ARCH == x86_64 ]]; then
```

Comment:
@aireilly I've removed this bit to always use `latest` unless `aarch64` or `arm64`. This is one difference from my previous PR.